test/plugins/similar-context-test.js

```diff
@@ -1,25 +1,86 @@
 const buster = require('buster')
+const stringSimilarity = require('string-similarity')
 const plugin = require('../../lib/plugins/similar-context-plugin.js')
 
 buster.testCase('similar-context-plugin', {
-  'should return 1.0 when the keywords of file description and task description match exactly': function () {
+  'test1 - with Keywordsextraction: should return 1.0 when both strings matches': function () {
     let sString1 = 'president obama woke monday facing congressional defeat parties believed hobble presidency'
     let sString2 = 'President Obama woke up Monday facing a Congressional defeat that many in both parties believed could hobble his presidency.'
-    let result = plugin(sString1, sString2)
+    let keywordExtraction = { extractKeywords: true }
+    let result = plugin(sString1, sString2, keywordExtraction)
     buster.assert.near(1.0, result, 0)
```

Comment:
In dem Fall mag es klappen, weil der Algorithmus exakt 1.0 zurück gibt. Wenn der Algo aber z.b. 0.99999 zurück gibt wird der Test fehlschlagen, wobei das auch nur Rundungsfehler sein könnte.